utron.go

```diff
@@ -213,8 +213,8 @@ func RegisterModels(models ...interface{}) {
 }
 
 // RegisterController registers a controller in the global utron App.
-func RegisterController(ctrl Controller, middlewares ...interface{}) {
-	_ = baseApp.router.Add(ctrl, middlewares...)
+func RegisterController(ctrlfn func() Controller, middlewares ...interface{}) {
```

Comment:
Can we improve this a little bit. This is going to break the API.

Maybe wrap the ctrl to a func that returns a new instance of ctrl everytime it is called?

This will keep everything  working and fix the issue altogether.